src/ints/bios_disk.cpp

```diff
@@ -164,10 +164,10 @@ uint8_t imageDisk::Read_Sector(uint32_t head,uint32_t cylinder,uint32_t sector,v
 
 uint8_t imageDisk::Read_AbsoluteSector(uint32_t sectnum, void *data)
```

Comment:
Is it possible to take `sector_num` in as a `cross_off_t`? 

(Maybe it ripples into some class API hierarchy.. if so, don't bother).

Reply:
No, sector number is not a direct offset. While in the case of raw disk images, they are directly related (offset is sector number * sector size), other formats that support sparse images may not be.

Reply:
Right; I was thinking of using `cross_off_t` to upgrade it to 64-bit at the API level, but didn't catch that the name itself ("off_t") is meant for offsets and this use would confuse the purpose.